src/jit/compiler.cpp

```diff
@@ -2194,6 +2194,16 @@ void Compiler::compDoComponentUnitTestsOnce()
         BitSetSupport::TestSuite(getAllocatorDebugOnly());
     }
 }
+
+unsigned char Compiler::compGetJitDefaultFill()
+{
+    unsigned char defaultFill = (unsigned char)JitConfig.JitDefaultFill();
+
+    // compStressCompile(STRESS_GENERIC_VARN, 50);
+
```

Comment:
To add random defaultFill values with JitStress I will add:

```

    if (compStressCompile(STRESS_GENERIC_VARN, 50))
    {
        defaultFill = info.compMethodHash();
        defaultFill ^= (defaultFill >> 12);
        defaultFill &= 0xff;
    }
```

Reply:
That will make it such that 50% time we pick a semi-random JitFill value based upon the Method Hash.

Reply:
Well we havethe lame assert around the JIT thatmake thsi difficult:

See morph.cpp line 2624:

```
void fgArgInfo::RecordStkLevel(unsigned stkLvl)
{
    assert(!IsUninitialized(stkLvl));
    this->stkLevel = stkLvl;
}

```